src/main/java/com/metamx/metrics/JvmMonitor.java

```diff
@@ -114,4 +127,28 @@ public boolean doMonitor(ServiceEmitter emitter)
 
     return true;
   }
+
+  private void emitGcMetrics(ServiceEmitter emitter)
+  {
+    ImmutableMap<String, Long> newGcCounters = ImmutableMap.of(
+      "jvm/gc/young/cpu", youngGcCpu.getNanos(),
```

Comment:
Do you think emitting just `jvm/gc/cpu` and `jvm/gc/count` as metrics name with `young`/`old` as one of dimensions will be better?

Also maybe it makes sense to keep old names of these metrics (`jvm/gc/time` and `jvm/gc/count`).